sky/clouds/service_catalog/images/provisioners/cuda-azure-grid.sh

```diff
@@ -11,6 +11,11 @@ wget -nv $GRID_DRIVER_URL -O $GRID_DRIVER_FILE
 sudo chmod +x $GRID_DRIVER_FILE
```

Comment:
nit: it seems we have two installation script in the home dir on the VM created with this image. let's remove them:
```bash
NVIDIA-Linux-x86_64-535.161.08-grid-azure.run  cuda_12.2.0_535.54.03_linux.run
```